api/src/routes/travCom-router.ts

```diff
@@ -5,7 +5,7 @@ import logger from "@/utils/logger"
 import { RequiresAuth } from "@/middleware"
 import { airports } from "@/json/airportCodes"
 import { TRAVCOM_DB_CONFIG } from "@/config"
-import { Location } from "@/models"
+import { FlightReconciliation, Location } from "@/models"
```

Comment:
I can probably delete most of this file.

Reply:
Nope, only 1/3 of it. I won't be able to remove this until I've cleaned up several other places in the app.